store/tikv/txn.go

```diff
@@ -357,6 +359,7 @@ func (txn *tikvTxn) LockKeys(ctx context.Context, lockCtx *kv.LockCtx, keysInput
 	if len(keys) == 0 {
 		return nil
 	}
+	keys = deduplicateKeys(keys)
```

Comment:
Is it necessary after https://github.com/pingcap/tidb/pull/16752/files#diff-b9e75a99170ea35a67074d5d3d3305adR3262 changed?

Reply:
It's the second line of defense.

Reply:
But it hurts the performance.

Reply:
the proportion would be tiny.

Reply:
ok